### ISSUE TITLE
Axe runner doesn't include wcag2aaa tag when standard is set to WCAG2AAA.

### ISSUE THREAD

**Issue by osimblahan** [pa11y/pa11y#666]
## Expected behaviour
Axe WCAG2AAA tagged tests should be included.

## Actual behaviour
They are not. (Only three tests exist at this time.)

Tag is listed in the axe docs. (There is also the wcag22aa tag with one test, but that's not technically a standard yet.)
https://www.deque.com/axe/core-documentation/api-documentation/#api-notes

## Steps to reproduce
Run pa11y with  `standard: 'WCAG2AAA'` and the axe runner.

## Problem area in runners/axe.js
```javascript
function pa11yStandardToAxe() {
    switch (options.standard) {
        case 'WCAG2A':
            return {
                type: 'tags',
                values: ['wcag2a', 'wcag21a', 'best-practice']
            };
        case 'WCAG2AA':
        case 'WCAG2AAA':
        default:
            return {
                type: 'tags',
                values: ['wcag2a', 'wcag21a', 'wcag2aa', 'wcag21aa', 'best-practice']
            };
    }
}
```

**Comment by camtin** [comment]
Since WCAG 2.2 is now the standard, I wonder if we should include WCAG 2.2.

Additionally, I wonder if we should "accept" options that are more like:

WCAG2A
WCAG21A
WCAG22A
WCAG2AA
WCAG21AA
WCAG22AA
WCAG2AAA
WCAG21AAA
WCAG22AAA

Or is there a specific reason Pa11y doesn't differentiate between the different revisions here?

**Comment by camtin** [comment]
Essentially, I think we want this (based on the available options from Axe core)
https://www.deque.com/axe/core-documentation/api-documentation/

```javascript


/**
--
  |   | * Map the Pa11y standard option to the aXe runOnly option.
  |   | * @private
  |   | * @returns {RunOnly} Returns the aXe runOnly value.
  |   | */
  |   | function pa11yStandardToAxe() {
  |   | console.log("Scan Standard: "+options.standard);
  |   | switch (options.standard) {
  |   | case 'WCAG2A':
  |   | return {
  |   | type: 'tags',
  |   | values: ['wcag2a','best-practice']
  |   | };
  |   |  
  |   | case 'WCAG2AA':
  |   | return {
  |   | type: 'tags',
  |   | values: ['wcag2a', 'wcag2aa', 'best-practice']
  |   | };
  |   |  
  |   | case 'WCAG2AAA':
  |   | return {
  |   | type: 'tags',
  |   | values: ['wcag2a', 'wcag2aa', 'wcag2aaa', 'best-practice']
  |   | };
  |   |  
  |   | case 'WCAG21A':
  |   | case 'WCAG22A':
  |   | return {
  |   | type: 'tags',
  |   | values: ['wcag2a', 'wcag21a', 'best-practice']
  |   | };
  |   |  
  |   | case 'WCAG21AA':
  |   | return {
  |   | type: 'tags',
  |   | values: ['wcag2a', 'wcag21a', 'wcag2aa', 'wcag21aa', 'best-practice']
  |   | };
  |   |  
  |   | case 'WCAG21AA':
  |   | return {
  |   | type: 'tags',
  |   | values: ['wcag2a', 'wcag21a', 'wcag2aa', 'wcag21aa', 'wcag2aaa', 'best-practice']
  |   | };
  |   |  
  |   | case 'WCAG22AA':
  |   | return {
  |   | type: 'tags',
  |   | values: ['wcag2a', 'wcag21a', 'wcag2aa', 'wcag21aa', 'wcag22aa', 'best-practice']
  |   | };
  |   |  
  |   | case 'WCAG22AAA':
  |   | return {
  |   | type: 'tags',
  |   | values: ['wcag2a', 'wcag21a', 'wcag2aa', 'wcag21aa', 'wcag22aa', 'wcag2aaa', 'best-practice']
  |   | };
  |   |  
  |   | default:
  |   | return {
  |   | type: 'tags',
  |   | values: ['wcag2a', 'wcag21a', 'wcag2aa', 'wcag21aa', 'wcag22aa', 'best-practice']
  |   | };
  |   | }
  |   | }


```

**Comment by aarongoldenthal** [comment]
> @danyalaytekin @osimblahan @camtin axe now includes `wcag22aa` in their API. Would it be possible to get it added in?

@kfairris I'm not suggesting there isn't room for improvement here, but `pa11y` v7.0.0 did upgrade `axe-core` from v4.2.1 to v4.8.2, which added the WCAG 2.2 rules (one new rule `target-size`, [in v4.5.0](https://www.deque.com/blog/axe-core-4-5-first-wcag-2-2-support-and-more/)). The default `pa11y` behavior is to not filter `axe` rules unless a `standard` is specified (via CLI or config), so without it the 2.2 rule is included.

For example, if you run `pa11y --runner axe https://www.w3.org/WAI/demos/bad/before/home.html`, you'll see in the results:

```sh
 • Error: All touch targets must be 24px large, or leave sufficient space (https://dequeuniversity.com/rules/axe/4.10/target-size?application=axeAPI)  
   ├── target-size
   ├── #mnav > ul > li:nth-child(2) > div > ul > li:nth-child(1) > a:nth-child(3)
   └── <a href="./reports/home.html" class="report"><span class="hidden">Inaccessib...</a>

 • Error: All touch targets must be 24px large, or leave sufficient space (https://dequeuniversity.com/rules/axe/4.10/target-size?application=axeAPI)  
   ├── target-size
   ├── #mnav > ul > li:nth-child(2) > div > ul > li:nth-child(2) > a:nth-child(3)
   └── <a href="../after/reports/home.html" class="report"><span class="hidden">Accessible...</a>
```

**Comment by josebolos** [comment]
Potentially improved by #685.
